Makefile.package

```diff
@@ -151,9 +160,28 @@ info-channel:
 info-fqn:
 	echo ${PACKAGE_FQN}
 
+info: parse-info
+	echo ${PACKAGE_INFO}
```

Comment:
When `$LOCKFILE_SOURCE` is not set, this will frequently leads to empty output since error messages e.g. due to version conflicts will be silently ignored.

Reply:
Errors and warnings are now output.